sceptre/stack.py

```diff
@@ -205,90 +192,66 @@ def __init__(
 
         self.hooks = hooks or {}
 
+    def _ensure_boolean(self, config_name: str, value: Any) -> bool:
+        if not isinstance(value, bool):
+            raise InvalidConfigFileError(f"Value for {config_name} must be a boolean, not a {type(value).__name__}")
```

Comment:
would be helpful if this provided an indication of which file contains the invalid configuration because this can happen in multiple files.

Reply:
Good point! I can add that.

Reply:
This has been done.